<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ISOTP_InterpreterWindow</class>
 <widget class="QDialog" name="ISOTP_InterpreterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>725</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ISO-TP Decoder</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Interpreted Messages Overview</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableIsoFrames"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="btnCaptured">
       <property name="text">
        <string>Interpret Previously Captured Frames</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClearList">
       <property name="text">
        <string>Clear Messages List</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="QCheckBox" name="cbShowIncomplete">
     <property name="text">
      <string>Show incomplete and/or corrupted messages</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="QCheckBox" name="cbUseExtendedAddressing">
     <property name="text">
      <string>Use extended addressing</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Data Details of Selected Message</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3"/>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="txtFrameDetails">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="lblFilter">
         <property name="text">
          <string>IDs to Decode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listFilter">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btnAll">
           <property name="text">
            <string>All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnNone">
           <property name="text">
            <string>None</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
